debian/control

```diff
@@ -3,12 +3,16 @@ Section: libs
 Priority: optional
 Maintainer: Ubuntu Developers <ubuntu-devel-discuss@lists.ubuntu.com>
 Rules-Requires-Root: no
-Build-Depends:
- debhelper-compat (= 13),
+Build-Depends: debhelper-compat (= 13),
  cmake,
- pkgconf,
+ doxygen,
+ gcc,
+ graphviz,
  libudev-dev,
  libmetee-dev,
+ make,
+ pkgconfig,
```

Comment:
This should be pkg-config

Reply:
this "issue" can be easily fixed, but in oracular it takes both, since oracular was the first version of ubuntu that has this change...